packages/now-cli/src/util/deploy/get-deployment-by-id-or-host.ts

```diff
@@ -69,7 +67,9 @@ async function getDeploymentByHost(
   apiVersion: APIVersion
 ) {
   const response = await client.fetch<Response>(
-    `/v4/now/hosts/${encodeURIComponent(host)}?resolve=1&noState=1`
+    `/v10/now/deployments/get?url=${encodeURIComponent(
+      host
+    )}&resolve=1&noState=1`
   );
-  return getDeploymentById(client, response.deployment.id, apiVersion);
+  return getDeploymentById(client, response.id, apiVersion);
```

Comment:
I think you can return `response` instead of calling `getDeploymentById()`.
```suggestion
  return response;
```

Also, remove `&resolve=1&noState=1` I think 🤔

Reply:
I guess we need to keep this as-is to support Now 1.0 deployments which use an older version of api-deployments.

Reply:
We will keep this as is until v1 is completely removed, because `limits` is only returned for v1 deployments when used the `v5` Deployments API.